posthog/models/filters/mixins/funnel_window_days.py

```diff
@@ -0,0 +1,15 @@
+from posthog.constants import FUNNEL_WINDOW_DAYS
+from posthog.models.filters.mixins.base import BaseParamMixin
+from posthog.models.filters.mixins.utils import cached_property, include_dict
+
+
+class FunnelWindowDaysMixin(BaseParamMixin):
+    @cached_property
+    def funnel_window_days(self) -> int:
+        _days = int(self._data.get(FUNNEL_WINDOW_DAYS, 14))
+        return _days
+
```

Comment:
you'll want to add an @include_dict decorated function so that this parameter is included in the cache hashes when converting the filter object to a dictionary to generate hash key. (Can use another filter param as reference) 

```
@include_dict
def funnel_window_days_to_dict(self):
...
```